client/src/components/map.tsx

```diff
@@ -112,8 +171,105 @@ const MapWrapper = ({features}: IMapWrapperProps) => {
     }
   }, [features]);
 
+  const handleMapClick = (event: { pixel: any; }) => {
+    const clickedCoord = mapRef.current.getCoordinateFromPixel(event.pixel);
+
+    mapRef.current.forEachFeatureAtPixel(event.pixel, (feature) => {
+      setSelectedFeature(feature);
+      feature.getProperties()['feature-state'] = 'selected';
+      //   let feature = new ol.Feature(new ol.geom.Point(coords));
+      // feature.setStyle(this.createIconStyle(this.coffeeShopIconPath, undefined));
+      // feature.setProperties({ name: 'test point', value: 15 });
+
+      // this.vectorSource.addFeature(feature);
+
+      return true;
+    });
+
+    overlayRef.current.setPosition(clickedCoord);
+  };
+
+  const readablePercent = (percent: number) => {
+    return `${(percent * 100).toFixed(2)}%`;
+  };
+
+  let properties;
+  if (selectedFeature) {
+    properties = selectedFeature.getProperties();
+  }
+
+  const getCategorization = (percentile: number) => {
+    let categorization = '';
+    if (percentile > 0.75 ) {
+      categorization = 'Prioritized';
+    } else if (0.60 < percentile && percentile < 0.75) {
```

Comment:
nice choice for formatting. 
Should one of these choices be >= ? (e.g. what if percentile is exactly 0.75)

Reply:
Good catch! will update